src/main/Organization/Organization.java

```diff
@@ -304,4 +307,62 @@ public int hashCode() {
     result.remove("id");
     return result;
   }
+
+  public JSONObject serialize() {
```

Comment:
is there some better way to serialize? As it stands, the current process involves when adding a new field, we have to remember to add a serialization for that, and bad things could probably happen if we forget to do that. Maybe we can make some serialization tests to match fields? I would like it to be platform-wide though

Reply:
Yeah I agree, this felt pretty clunky and it would be great to have testing in place at the least. I'm not sure if there's a better pattern in Java for doing this sort of transformation in a more automated way.